apps/system/js/bluetooth_transfer.js

```diff
@@ -644,15 +644,17 @@ var BluetoothTransfer = {
     var storage = navigator.getDeviceStorage(storageType);
     var getreq = storage.get(filePath);
```

Comment:
I assume this is a DOMRequest. If so it's a thenable. So better way to do this would be:

```javascript
var getreq = storage.get(filePath);
var loader = LazyLoader.load(['shared/js/mime_mapper.js']);
Promise.all([getreq, loader])
  .then(<success>, <error>)
  .catch(...);
```

The `then()` block would ensure both thenables is resolved and the `.catch()` would catch throw from both functions.